crates/bevy_input/src/gamepad.rs

```diff
@@ -505,6 +531,34 @@ impl Gamepad {
             .into_iter()
             .all(|button_type| self.just_released(button_type))
     }
+
+    /// Returns an iterator over all digital [button]s that are pressed.
+    ///
+    /// [button]: GamepadButton
+    pub fn get_pressed(&self) -> impl Iterator<Item = &GamepadButton> {
```

Comment:
Non-blocking bikeshed question: Should these iterator methods instead be called `iter_pressed`, `iter_just_pressed` etc.?

Reply:
We should be consistent with the ButtonInput API: https://docs.rs/bevy/latest/bevy/input/struct.ButtonInput.html#method.get_pressed

Reply:
Perfectly reasonable!